content/en/blog/posts/2023-11-07-beta-api.md

```diff
@@ -0,0 +1,146 @@
+
+We strongly encourage both current and future users to adopt the _beta_ API to benefit from
+its enhanced definition.
+
+
+## Migration guidelines
+
+### Deprecated fields
+
+| Resource |  Field                           | Alternative                             |
+|----------|----------------------------------|:---------------------------------------:|
+| Build    | `.spec.sources`                  | `.spec.source`                          |
+| Build    | `.spec.dockerfile`               | `spec.paramValues[]` with `dockerfile`  |
+| Build    | `.spec.builder`                  | none                                    |
+| Build    | `.spec.volumes[].description`    | none                                    |
+| BuildRun | `.spec.serviceAccount.generate`  | `.spec.serviceAccount` with `.generate` |
+| BuildRun | `.spec.sources`                  | `.spec.source`                          |
+| BuildRun | `.spec.volumes[].description`    | none                                    |
+
+
+### Changes to Build API fields
+
+|  Old field                       |                            New field                                     |
+|----------------------------------|:------------------------------------------------------------------------:|
+| `.spec.source.url`               |  `.spec.source.git.url`                                                  |
+| `.spec.source.bundleContainer`   |  `.spec.source.ociArtifact`                                              |
+| `.spec.sources` for `LocalCopy`  |  `.spec.source.local`                                                    |
+| `.spec.source.credentials`       |  `.spec.source.git.cloneSecret` or `.spec.source.ociArtifact.pullSecret` |
+| `.spec.output.credentials`       |  `.spec.output.pushSecret`                                               |
+
+
+See this example of the Git source type:
+
+```yaml
+# v1alpha1
+---
+apiVersion: shipwright.io/v1alpha1
+kind: Build
+metadata:
+  name: a-build
+spec:
+  source:
+    url: https://github.com/shipwright-io/sample-go
+    contextDir: docker-build
+  strategy:
+    name: buildkit
+    kind: ClusterBuildStrategy
+  output:
+    image: an-image
+
+# v1beta1
+---
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: a-build
+spec:
+  source:
+    type: Git
+    git: 
+      url: https://github.com/shipwright-io/sample-go
+    contextDir: docker-build
+  strategy:
+    name: buildkit
+    kind: ClusterBuildStrategy
+  output:
+    image: an-image
+```
+
+
+### Changes to BuildRun API fields
+
+|  Old field                  |                 New field              |
+|-----------------------------|:--------------------------------------:|
+| `.spec.buildSpec`           | `.spec.build.spec`                     |
+| `.spec.buildRef`            | `.spec.build.name`                     |
+| `.spec.sources`             | `.spec.source` only for `Local`        |
+| `.spec.serviceAccount.name` | `.spec.serviceAccount` with `.generate`|
```

Comment:
I think the intent here is to state that `.spec.serviceAccount.generate` is gone?

```suggestion
| `.spec.serviceAccount.generate` | `.spec.serviceAccount` with `.generate`|

_Note: generated service accounts is a deprecated feature, and may be removed in a future release._
```

Reply:
I like that additional sentence.